Atlas/frontend/src/containers/CulturalHeritage/Add.js

```diff
@@ -73,7 +68,8 @@ const mapDispatchToProps = dispatch => {
         headers: { 'Authorization': 'JWT ' + token },
         data: {
           title: addCHInputs.title,
-          description: addCHInputs.description
+          description: addCHInputs.description,
+          tags: addCHInputs.tags.map(t => ( { name: t.text } ))
```

Comment:
![image](https://user-images.githubusercontent.com/23107192/32729060-9cf30d7c-c893-11e7-92da-1cc447bd75cb.png)
I am getting an error here

Reply:
Can you show the error message? It's in the upper part of your screen shot, and missing :/